src/main/java/ToDo.java

```diff
@@ -0,0 +1,10 @@
+public class ToDo extends Task {
```

Comment:
Good that class name follows upper camel case, also good use of inheritance to prevent repetitive code